styles/electric.json

```diff
@@ -34,6 +34,11 @@
 	"styles": {
 		"elements": {
 			"button": {
+				"border": {
+					"style": "solid",
+					"width": "1px",
```

Comment:
2px is the default for outline button. It is best to define the `2px` border. update the same for hover state.

<img width="335" alt="image" src="https://user-images.githubusercontent.com/1935113/192757820-69cb0a26-40bd-49cf-83ee-eaa022f618f2.png">

Reply:
Thank you for the suggestion @madhusudhand. I have update the border thickness.